docs/source/learn/trace-vs-execution-time.md

```diff
@@ -0,0 +1,144 @@
+# Tracing Time vs. Execution Time in PyTorch/XLA
+
+When working with PyTorch/XLA, it's essential to understand that operations on
+XLA tensors are not typically executed immediately in the way they are with
+standard PyTorch tensors on CPU or CUDA devices (which operate in "eager mode").
+PyTorch/XLA employs a "lazy execution" model. This means that when you write
+PyTorch code using XLA tensors, you are primarily defining or tracing a
+computation graph. The actual execution of this graph on the XLA device (like a
+TPU) is deferred until a specific trigger point.
+
+This leads to two distinct types of "time" to consider:
+
+1. Tracing Time: The time spent by PyTorch/XLA to record operations, build the
+computation graph.
+1. Execution Time: The actual time the XLA device spends compiling (if
```

Comment:
do we want to split "execution time" to "compile time" + "execution time"?

Reply:
makes sense. Done.